packages/lexure/tests/lexer.test.ts

```diff
@@ -0,0 +1,54 @@
+import { Lexer, QuotedParameter, WordParameter } from '../src';
+
+describe('The lexer tests', () => {
+	it('GIVEN input with no quotes THEN return the string as is', () => {
+		const input = 'text without quotes';
+		const output = [...new Lexer().run(input)];
```

Comment:
Please don't use `output`, use `result`, and separate the initialization from the assertions. Apply this everywhere else as well.